frontend/.env

```
@@ -0,0 +1 @@
+VITE_BACKEND_URL=hpocketplans-production.up.railway.app
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)